External/SDK/Clang/Windows/Clang10.bff

```diff
@@ -1,6 +1,6 @@
 // Clang 10.x.x
 //------------------------------------------------------------------------------
-.Clang10_BasePath     = '$_CURRENT_BFF_DIR_$/10.0.0.RC3'
+.Clang10_BasePath     = 'C:\Program Files\LLVM'
```

Comment:
It looks like you made some changes to use your local installation of Clang and MSVC.

These changes should be reverted.

Reply:
Yes, shame on me.

Reply:
Reverted in 597fed9b0a6242f837605d745cd451a684653c10.